lib/src/model.dart

```diff
@@ -1121,6 +1121,21 @@ class ModelFunction extends ModelElement {
 
   @override
   String get _href => '${library.dirName}/$fileName';
+
+  String get sourceCode {
```

Comment:
We should make a common method out of this and the below one. We should also take the common indent (or at least the indent of the first line) and remove it from all the source lines.